server/src/storage/segment.rs

```diff
@@ -128,14 +121,15 @@ impl UncompressedSegment for SegmentBuilder {
 
     /// Spill the in-memory segment to an Apache Parquet file and return Ok when finished.
     fn spill_to_apache_parquet(&mut self, key: String) -> Result<SpilledSegment, std::io::Error> {
```

Comment:
Maybe import `std::io::error` as `IOErrro` to differentiate it from `ParquetError`.

Reply:
Fixed.